ivysettings.xml

```diff
@@ -0,0 +1,22 @@
+<ivysettings>
+  <!-- Properties to be used in ivy.xml -->
+  <property name="opensaml.version" value="3.3.0" />
```

Comment:
This doesn't seem to be used, and ivy.xml refers to version 2.6.6?

Reply:
Indeed, removed. Also, to answer your initial question, the motivation behind introducing Ivy was to prepare for OpenSAML 2->3 migration (2.x is deprecated / security-unsupported).